src/redux/migrations.ts

```diff
@@ -793,15 +793,28 @@ export const migrations = {
     },
   }),
   72: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      showGuidedOnboardingCopy: REMOTE_CONFIG_VALUES_DEFAULTS.showGuidedOnboardingCopy,
```

Comment:
you should add a new migration, not modify an old one. You can technically get away with modifying an existing migration-- safest one is probably the last one-- IF your modification goes out in the *same release* as the code that added that migration to begin with. But it's always safer to just add a new one.

Reply:
please undo all your changes to existing migrations in this file and add a new one to include your addition of this feature flag.